### DISCUSSION TITLE
error please install git client

### DISCUSSION THREAD

**Discussion by n9jcv** [community/community#21731] [New to GitHub]
<p>Sorry another newbie question, I have platformio and github desktop</p>
<p>I downloaded a zip of Marlin 1.1.8 and created a repository.  I then opened the repository in platformio.  I click platformio build and then I get this message;</p>
<p>LibraryManager: Installing LiquidTWI2<br>
Error: Please install Git client from <a href="https://git-scm.com/downloads" rel="nofollow noopener">https://git-scm.com/downloads</a><br>
I first went to install packages in github desktop and searched liquidtwi2 - it could not be found.  The error message also says to install Git client.  I have github desktop, why does it not recognize that.  </p>
<p>Can anyone please help me get past these issues?<br>
Thank You</p>

**Comment by n9jcv** [comment]
<p>for clarification, when I say platformio, I guess it is really Atom</p>

**Comment by n9jcv** [comment]
<p>Well I don’t know exactly why or how, but I seem to have gotten past it. </p>
<p>I deleted my repository, both at Github and in Github desktop and Atom.</p>
<p>Then I went back and found Marlin 1.1.8 and did a fork.  Then I went to Github desktop and cloned the fork.  Then opened the fork in Atom and now it works.  </p>
<p>So it really did not have anything to do with the Git client, as I had it before, and still have it now and did not change anything.  I just wish I knew what was wrong???</p>

**Comment by n9jcv** [comment]
<p>There is a problem with this though.  I was on the release page for 1.1.8 and clicked fork, but instead it gave me a fork of 1.1.9.  This ties in with my other post.  I want a fork of 1.1.8 but I cant seem to find out how to get it.</p>
<p>Thanks Again</p>

**Comment by n9jcv** [comment, marked as answer]
<p>Ok, I finally figured this out, </p>
<p>I have the Github Desktop already installed.  The error message is for the Git Client though and NOT the desktop.  So I downloaded the Git Client.  Then I had to remove my project library, like twice and reboot, but eventually, it found the Git Client, and now I can build and upload.  I hope this helps someone in the future.</p>

**Comment by ghost** [comment]
<p>I had the same problem, but now everything is working, apparently the developers have fixed the error</p>
